std/experimental/ndslice/slice.d

```diff
@@ -567,6 +567,19 @@ slice(T,
 }
 
 /// ditto
+auto slice(T,
+    Flag!`replaceArrayWithPointer` replaceArrayWithPointer = Yes.replaceArrayWithPointer,
+    size_t N)(auto ref in size_t[N] lengths, auto ref T init)
```

Comment:
turns out Allocator uses the same template:
https://github.com/dlang/phobos/blob/master/std/experimental/allocator/package.d#L681

```
T[] makeArray(T, Allocator)(auto ref Allocator alloc, size_t length,
    auto ref T init)
```